### DISCUSSION TITLE
So why was everything about my all features wiki edit reverted? I'm open to suggestions

### DISCUSSION THREAD

**Discussion by Justformas** [Monika-After-Story/MonikaModDev#10340] [general]
My edit did a few different things so to break it down:

- Mentioned that affection wasn't intended to be a visual representation of your progress, as this was a suggestion from a developer. Also mentioned that not all progress was made through it to de-emphasize it, so people aren't mislead to think that it's the main purpose/feature of the mod. Also because that's the truth; the mod has other systems related to progress, but that's not what that section is about, so I didn't go into details. But this was reverted.

- Made a note in the "how to view your affection" section that logs are intended for debugging information rather than for players to track progress. Something like this has also been said by the developers many times in issues/discussions, but there was no mention of that on this wiki page that many players reference. I felt that a note was necessary there to make it clear that this "how to" section wasn't something intended as a part of the intended experience for players by the developers. Also in case people skip reading the first part about affection and just went straight to that section, didn't read it thoroughly or forgot. It's like another sort of warning so that players can make a more informed decision about deciding whether to view it anyway.
But this too was reverted.

- I had removed the 4 different action groups about affection since they are now inaccurate/outdated and there are more of them now. While I could have included all the groups with updated numbers for standard gain/loss events, I didn't think that was necessary, and no one seemed to say anything about it when I brought this up. But this too was reverted, back to the outdated information.

- Updated the daily cap and bypasses section which was also inaccurate/outdated. If anything that I updated there was wrong, then you can say so, but I didn't think it was. I also intentionally was not too specific about some of the exact values, but I could be if that would be preferred. Also added a note about a newer bypassed affection mechanic that people had ask related questions about. But all of this was also reverted, back to the outdated information.

- I had updated the number to reach the "happy" level of affection, as I thought that was something that was changed a few years ago, but if I was wrong about this then I'd like to know, because this was also reverted.

- Updated information on whether the Christmas Cookies and Candy Cane gifts bypass to true. I'm pretty sure they both do, more than once (doesn't necessarily mean unlimited). But this was reverted to First time Given and False respectively.

So I want to know why all of this was reverted. I don't think all of it was wrong or that all of the reverted outdated information is more correct. I could edit just one thing at a time, if that is preferred. But equally as important, after how much I've heard that affection isn't for the player to view or to be a visual representation of progress, is it really better to just mention affection and how to view it, without saying anything about that not being an intended feature for players? On a page with the word "features" in its title? This is the sort of thing that causes people to believe that it's a feature. I'm still for letting people decide for themselves, but I want people to know the developers' intent as well. But, if it so happens that some developers are split on this, then maybe it's better to not say anything about developer intent. But I thought there was a sort of "intended" way to play regarding this and it'd be nice if that could be reflected on that page, and players can decide whether they want to follow that or not, with that information in mind.

**Comment by ThePotatoGuy** [comment]
someone vandalized the wiki page so @Booplicate reverted the vandalism and force pushed to erase the history. I assume any reversions of actual commits was not intentional, but I'll wait for him to confirm before reverting the revert.

**Reply by Justformas** [reply]
Thanks for the quick response PotatoGuy, especially considering that I haven't seen you here much lately. I don't suppose you know who vandalized that page? Understandably the deletion might've had to have been erased from or not even show up in history, but don't know why my edit, which happened before that, would've had to have been reverted as well, especially with it still in the history. But maybe it's just weird like that. If it was an unintentional revert though, then it shouldn't take long to resolve.

But if it was intentional, I don't know why having well outdated information and affection being presented in a way that may lead players to believe that it and viewing it are an intended feature for them is supposedly better. Per the policy that's been on that page, it just says to "please keep content in the current release out of here until 1 release version has passed. (Aka, 0.8.5 stuff can be added here after 0.8.6 is released)". We've been at 0.12.15 for awhile and everything in my edit has been the case for the mod since 0.12.10 or even sooner in some cases, so it should be fine to add or at least update that outdated information by now.

Well when or if a decision is reached, let me know, or can just revert if appropriate.

**Reply by ThePotatoGuy** [reply]
I banned the 2 users from the repo. 

I reverted the commit, so let me know if the current iteration seems accurate.

**Reply by Justformas** [reply]
2 users were involved? This whole thing was rather strange, but great that you could figure that out and take care of it.

As far as I can tell, the current iteration looks exactly as I left it, thanks. 

Of course, the wording could always be changed if it could be better. It was also an attempt to clean some stuff up. By the way, I guess it's not too surprising, but was still interesting for me to learn that it was you who had initially created that affection section for people to put details in.

**Comment by Booplicate** [comment]
@ThePotatoGuy,
1. I do not want it to be updated to accurately describe the system
2. There were mistakes in the commit

both of those lead to misunderstanding and gamifying, which we've seen over the last few years.

@Justformas you in fact knew that I planned to nuke/rewrite that part, not sure why you're surprised.

**Reply by Booplicate** [reply]
I did not erase it from the commit history because some of it could be useful later. I only nuked the commits from 2 users which vandalised the page. Yours was reverted to be revisited later.

**Reply by Justformas** [reply]
It hadn't been updated for quite some time, and as I recall you said that you guys generally don't update that wiki. Between that and it's history, I thought that's why you hadn't changed it even if you didn't like how it was written. So no, I did not know that you had planned to rewrite that part.

The outdated information and way it was before leads to misunderstanding and gamifying anyway, because it is presented like a feature for players and is stated how to view it. At least stating the purpose of affection and logs might encourage some players to play the intended way that you want. You'd rather have no statement about that in the meantime?

Mistakes can be corrected, though I'm not sure what was wrong about it, and you don't seem to want to say and would apparently rather have it all reverted and outdated in the meantime. But even if it was all correct, you say you don't want it updated accurately anyway. That's sort of the point of a wiki though, to have accurate information. And on a page called "All Features [SPOILERS]" even providing details on how internal features work should be fine, and that section did so for most of that page's history. People should have the right to know if they seek it out. Yes, it may "spoil" the intended experience if you play a different way, but hence the warnings, and people can play how they want. Anyway, having to wait for one person to rewrite something and not being able to update old information in the meantime isn't how it should work.

**Reply by Booplicate** [reply]
> It hadn't been updated for quite some time, and as I recall you said that you guys generally don't update that wiki. Between that and it's history, I thought that's why you hadn't changed it even if you didn't like how it was written. So no, I did not know that you had planned to rewrite that part.

That is correct except you did knew a rewrite was incoming because I told that to you explicitly.

> The outdated information and way it was before leads to misunderstanding and gamifying anyway

If it was bad, it doesn't mean you should amplify it.

> would apparently rather have it all reverted and outdated in the meantime.

Well it was easier to keep as is, but I may as well nuke that part for now.

> That's sort of the point of a wiki though, to have accurate information. And on a page called "All Features [SPOILERS]" even providing details on how internal features work should be fine, and that section did so for most of that page's history.

Kind of, however we do not explain many others internal mechanics and I would rather not. So that page is inconsistent in general with what we do.

> people can play how they want

Sure, I just don't want *us* to encourage the unintended way that ruins the immersion. You also used that before to justify being able to see affection because we explain how to read logs and what affection is on our wiki. Another reason to nuke it.

**Reply by Justformas** [reply]
> That is correct except you did knew a rewrite was incoming because I told that to you explicitly.

No you said it _should_ be rewritten which is different. You also said something like that nearly 2 years ago yet nothing was done about it. How was I supposed to know that this time, that actually meant "I'm going to make an incoming rewrite, don't touch", if that even really is what you meant? That would be explicit, not how you said it.

And guess what, I was the one to take a stab at rewriting it. I even added what you suggested about it not being meant as visual representation of progress. But I was also trying to balance everything. Your wishes (hence warnings/statements and not all details about some of the exact cap values or action groups), as well as mine and people who like knowing that information, and not rocking the boat too much with the mod and info on that page.

> If it was bad, it doesn't mean you should amplify it.

Whether it was bad or my edit was worse is an opinion, and this sounds like you're saying that none of the info from my edit was good or possibly even correct, but I highly doubt that. I even did something that you said to but you call that bad? I'd argue that outdated information and lack of a statement about what those things are meant for is worse. Whether you think info about internal mechanics should be there or not.

> Well it was easier to keep as is, but I may as well nuke that part for now.

Maybe it was easier but how hard would it have been to only revert what you didn't like or was wrong?

> Kind of, however we do not explain many others internal mechanics and I would rather not. So that page is inconsistent in general with what we do.

Explaining other internal mechanics would be a bigger change and I feel that you developers could be the ones to add or start sections about that if you wanted to. But Potato had initially created that section about affection and though he didn't write the details, I was just updating what was there. 

You seem to be the only one so opposed to having this information about affection revealed, although I don't think I've ever heard @ThePotatoGuy 's opinion about this whole affection and log thing and would very much like to hear it. I don't think suppression of information especially that has been there for awhile is a good thing.

> Sure, I just don't want _us_ to encourage the unintended way that ruins the immersion. You also used that before to justify being able to see affection because we explain how to read logs and what affection is on our wiki. Another reason to nuke it.

Which is also why I put warnings/statements about what they are for, and it's been stated that the developers didn't write that wiki but rather the community. Anyway for people that have been playing this way for awhile and like having that information, it's unfair to alienate them. There should be able to be a balanced solution rather than just all of one person's wishes.
